backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/components/EdgeRoutingPointsProvider.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Thales.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.diagrams.components;
+
+import java.util.List;
+
+import org.eclipse.sirius.web.components.Element;
+import org.eclipse.sirius.web.diagrams.Position;
+
+/**
+ * Provides the routing points to apply to a new edge.
+ *
+ * @author fbarbin
+ */
+public class EdgeRoutingPointsProvider {
+    public List<Position> getRoutingPoints(Element source, Element target) {
+        return List.of();
```

Comment:
Why aren't you providing the routing points specified in the ADR?

Reply:
In a first version, the edge layout is properly displayed without any routing point. The result could be worst by setting two routing points corresponding to the source and target x,y location

Reply:
We want to have this information server-side. We don't want to rely on Sprotty for that. Compute this information here to create a node from the top left of the source to the top left of the target.

Reply:
Done in the commit "Compute Edge routing points"